logstash-codec-fluent.gemspec

```diff
@@ -22,11 +22,9 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
 
-  if RUBY_PLATFORM == 'java'
-    s.platform = RUBY_PLATFORM
-  end
+  s.platform = 'java'
   
-  s.add_runtime_dependency 'msgpack'
+  s.add_runtime_dependency 'msgpack', '= 1.1.0'
```

Comment:
maybe we can relax this a bit?

```ruby
s.add_runtime_dependency 'msgpack', '~> 1.1'
```

Reply:
@jsvd as a Java person I hate doing that, but I guess in Ruby it's fine :), sec relaxing it.

Reply:
done :)